iib/workers/tasks/build.py

```diff
@@ -484,6 +489,10 @@ def _skopeo_copy(source, destination, copy_all=False, exc_msg=None):
     ]
     if copy_all:
         args.append('--all')
+    if dest_token:
+        log.debug('Using user-provided token to copy the container image')
+        args.append('--dest-creds')
+        args.append(dest_token)
```

Comment:
`run_cmd` will need to be adjusted to not log the credentials in case it fails. It'll be similar to this:
https://github.com/release-engineering/iib/commit/41d101c08edd132a9a096857016049f0e0f61a93#diff-1c6ad2472cabab850bfc0c408419ccaaL74-L80

Reply:
Great catch! I implemented something similar to celery's argsrepr.